components/collaboration/collaboration-details/modal/ManagerModal.tsx

```diff
@@ -0,0 +1,97 @@
+import { Image, Pressable, Linking } from "react-native";
+import { useTheme } from "@react-navigation/native";
+
+import { Text, View } from "@/components/theme/Themed";
+import stylesFn from "@/styles/modal/UploadModal.styles";
+import { Chip, Modal } from "react-native-paper";
+import Colors from "@/constants/Colors";
+
+interface ManagerModalProps {
+  managerName: string;
+  managerEmail: string;
+  brandDescription: string;
+  managerImage: string;
```

Comment:
Instead of passing props like this, we can create an object of related property -
Example - 
```
manager: {
 name: string;
 email: string;
 image: string;
}
brand: {
 description: string;
}
```